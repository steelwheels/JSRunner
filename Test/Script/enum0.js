/* enum0.js */

const center = Align.center ;
const top    = Align.top ;
const red    = Color.green ;

console.log("center = " + center + "\n") ;
console.log("top    = " + top + "\n") ;
console.log("red    = " + red + "\n") ;
