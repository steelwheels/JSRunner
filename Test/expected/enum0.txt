center = 2
top    = 0
red    = 2
